### DISCUSSION TITLE
Using prisma2 but with my own SQL migrations.

### DISCUSSION THREAD

**Discussion by sticker221** [prisma/prisma#2381] [Q&A] [answered]
There are a bunch of postgres features I'd like to use but are not supported yet by prisma like 
table inheritance and partitioning, check constraints, json columns and others. 

Would it be possible to use prisma2 only to generate the client and leave it up for the user to maintain the database schema with their own migrations, so that we can use at least some of these features?

**Comment by johhansantana** [comment, marked as answer]
I think you could. Everytime you do the manual updates you could probably do a `prisma introspect` to get the latest database changes?

**Reply by janpio** [reply]
Yes, that is a perfectly viable usage strategy for Prisma.

**Comment by ludralph** [comment]
Hi @sticker221,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
